spec/features/issues_spec.rb

```diff
@@ -40,9 +40,9 @@
       it "should update issue fields" do
         click_button "Save changes"
 
-        page.should have_content @user.name
-        page.should have_content "bug 345"
-        page.should have_content project.name
+        expect(page).to have_content @user.name
+        expect(page).to have_content "bug 345"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.